pkg/k8s/apis/cilium.io/utils/utils.go

```diff
@@ -32,6 +32,9 @@ const (
 	// podPrefixLbl is the value the prefix used in the label selector to
 	// represent pods on the default namespace.
 	podPrefixLbl = labels.LabelSourceK8sKeyPrefix + k8sConst.PodNamespaceLabel
+	// podAnyPrefixLbl is the value the prefix used in the label selector to
```

Comment:
Nit: change to "is the value of the prefix used"

Also add a blank line before this.

Reply:
Done